CMakeLists.txt

```diff
@@ -80,9 +80,9 @@ if(NOT ${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 endif()
 
 # configure OTF2xx submodule
-set(OTF2XX_CHRONO_DURATION_TYPE nanoseconds)
+set(OTF2XX_CHRONO_DURATION_TYPE nanoseconds CACHE INTERNAL "")
```

Comment:
I cannot make a qualified comment about that. If you are confident about this, then fine.

Reply:
![image](https://user-images.githubusercontent.com/130405/44712434-05eab000-aab1-11e8-857b-40d9eafdde9d.png)

Don't ask.